src/Task_21/PersonService.java

```diff
@@ -0,0 +1,74 @@
+package Task_21;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.util.Iterator;
+import java.util.Set;
+import java.util.TreeSet;
+
+public class PersonService {
+
+    public static Set<Person> people = new TreeSet<>(new Comparator());
```

Comment:
It's realy not a good idea to have static variables for this task.
The better solition will be to have several methods for the next actions:
1. Read the file and return collection of people. (method 1)
2. Do some actions like count (method 2)
3. Write data to file (Method 3)
These new methods could be also static, but in the proposed solution we can use them independently